pkg/R/DataFrame.R

```diff
@@ -0,0 +1,119 @@
+# DataFrame.R - DataFrame class and methods implemented in S4 OO classes
+
+#' @include jobj.R RDD.R
+NULL
```

Comment:
Yea, this is how roxygen knows to correctly generate the collate field.  @include allows me to specify that jobj.R and RDD.R should both come before the DataFrame.R in the build older when using R CMD install.  This is essential for method dispatch, as DataFrame.R inherits from some of the S4 generics defined in RDD.R